src/HotChocolate/Caching/src/Caching/Types/CacheControlDirective.cs

```diff
@@ -5,16 +5,24 @@ public sealed class CacheControlDirective
     public CacheControlDirective(
```

Comment:
Before this PR I implemented Vary on the side using my own directive and attribute (I had to pull in my own middleware as I couldn't modify the DefaultHttpResponseFormatter). It worked well on the side, which begs the question if Vary should be treated separately or if it's best handled in the same directive, like this?